src/css/profile/mobile/common/popup.less

```diff
@@ -79,11 +102,47 @@
 			margin-bottom: -12 * @unit_base;
 			margin-right: 22 * @unit_base;
 		}
+
+		@media (orientation: portrait) and (max-width: 479px) {
+			padding: 0 * @px_base 24 * @px_base 24 * @px_base 24 * @px_base;
+		}
+		@media (orientation: landscape) and (max-height: 479px) {
+			padding: 0 * @px_base 24 * @px_base 12 * @px_base 24 * @px_base;
+		}
+		@media (orientation: portrait) and (min-width: 600px) {
+			padding: 0 * @px_base 24 * @px_base 24 * @px_base 24 * @px_base;
+		}
+		@media (orientation: landscape) and (min-width: 960px) {
+			padding: 0 * @px_base 24 * @px_base 12 * @px_base 24 * @px_base;
+		}
 	}
 
 	.ui-popup-footer {
 		background-color: @color_popup_footer_bg;
+		position: sticky;
 		.LESSui-footer();
+
+		@media (orientation: portrait) and (max-width: 479px) {
+			padding: 0 * @px_base 24 * @px_base 20 * @px_base 24 * @px_base;
+		}
+		@media (orientation: landscape) and (max-height: 479px) {
+			padding: 0 * @px_base 24 * @px_base 4 * @px_base 24 * @px_base;
+		}
+		@media (orientation: portrait) and (min-width: 600px) {
+			padding: 0 * @px_base 24 * @px_base 20 * @px_base 24 * @px_base;
+		}
+		@media (orientation: landscape) and (min-width: 960px) {
+			padding: 0 * @px_base 24 * @px_base 4 * @px_base 24 * @px_base;
+		}
+	}
+
+	.ui-listview {
+		border-top-width: 1 * @px_base;
+		border-top-color: #CCCCCC;
```

Comment:
Is it correct color for the border?

Reply:
This is the color I put in temporarily.
I plan to change it later when showing the divider dynamically as scroll.